plugins/solidigm/solidigm-id-ctrl.c

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * Copyright (c) 2023 Solidigm.
+ *
+ * Author: leonardo.da.cunha@solidigm.com
+ */
+
+#include <inttypes.h>
+#include "common.h"
+#include "solidigm-id-ctrl.h"
+
+#pragma pack(push, vu_id_ctrl, 1)
```

Comment:
Why not using the attribute `__packed` ?

Reply:
Done!